pybuildkite/client.py

```diff
@@ -50,13 +50,14 @@ def request(self, method, url, query_params=None, body=None, headers=None):
             body = self._clean_query_params(body)
 
         response = requests.request(method, url, headers=headers, params=query_params, json=body)
-        response.raise_for_status()
-
-        # TODO what should be returned
-        if headers:
+        try:
+            response.raise_for_status()
+        except:
```

Comment:
It looks like we're no longer going to raise an exception here which is a bit troubling. We'll want to raise the exception.

Reply:
I will revert it.